node/pkg/fetcher/app.go

```diff
@@ -396,6 +406,51 @@ func (a *App) initialize(ctx context.Context) error {
 	return nil
 }
 
+func (a *App) localAggregatesChannelProcessor(ctx context.Context, localAggregatesChannel chan LocalAggregate) {
```

Comment:
it is recommended to use verb for function names, when it ends with word such as processor, it sounds more like variable or structure.

how about `bulkStoreLocalAggregates` or something similar to that? I think `process` is quite having a broad meaning inside